### PR TITLE
feat : Added radio, checkbox and textarea inputs

### PR DESCRIPTION
**Pull request by pratik-waghmare**
__Features added__

- [x] Added radio input component
- [x]  Added checkbox input component
- [x]  Added textarea input component
- [x]  Created global context for user's answers
- [x]  Added loading animation with delay

__ChatBot Link__
[rc-chatbot](https://rc-chatbot.netlify.app/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import { ChatBox } from "./Component/ChatBox/ChatBox";
+import { ChatBotContext } from "./Context";
+
+const AppWrapper = styled.div``;
 
 function App() {
+  const [global, setGlobal] = useState<object>({});
```
Comment (digvijayu) [inline src/App.tsx]: Pls define object as an interface or type

File: src/Component/ChatBox/ChatBox.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+import { ChatBubble } from "../ChatBubble/ChatBubble";
+import { TextInput } from "../Inputs/TextInput/TextInput";
+import { TextAreaInput } from "../Inputs/TextAreaInput/TextAreaInput";
+import { RadioInput } from "../Inputs/RadioInput/RadioInput";
+import { CheckboxInput } from "../Inputs/CheckboxInput/CheckboxInput";
+import LoadingAnimation from "../LoadingAnimation/LoadingAnimation";
+
+import ContextModal from "../ContextModal";
+
+export interface ChatBoxProps {}
```
Comment (digvijayu) [inline src/Component/ChatBox/ChatBox.tsx]: No props?

File: src/Component/ChatBox/ChatBox.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+export const ChatBox = () => {
+  const [messageArray, setMessageArray] = useState<Array<Actor>>([]);
```
Comment (digvijayu) [inline src/Component/ChatBox/ChatBox.tsx]:
```suggestion
  const [messageArray, setMessageArray] = useState<Actor[]>([]);
```

File: src/Component/ChatBox/ChatBox.tsx [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+        />
+      );
+      break;
+
+    case "empty":
+      inputComponent = <div />;
+      break;
+
+    default:
+      inputComponent = (
```
Comment (digvijayu) [inline src/Component/ChatBox/ChatBox.tsx:122]: should raise an error..

File: src/Context.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { createContext } from "react";
+
+export interface ChatBotContextProps {
+  global: any;
```
Comment (digvijayu) [inline src/Context.tsx]: needs to have a shape, please don't use any anywhere

**Review by umangag07 — APPROVED** [review]

**Review by digvijayu — APPROVED** [review]

**Pull request merged**